frontend/src/components/FAQs.tsx

```diff
@@ -0,0 +1,342 @@
+import plusIcon from "../assets/plusIcon.svg";
+import { Accordion } from "../components/index.ts";
+import styles from "../stylesheets/Home.module.css";
+
+import { useState } from "react";
+
+export function FAQs() {
+  const [expandAllCertificationInfo, setExpandAllCertificationInfo] = useState(false);
+  const [expandAllPathwaysInfo, setExpandAllPathwaysInfo] = useState(false);
+
+  const toggleExpandAllCertificationInfo = () => {
+    setExpandAllCertificationInfo(!expandAllCertificationInfo);
+  };
+  const toggleExpandAllPathwaysInfo = () => {
+    setExpandAllPathwaysInfo(!expandAllPathwaysInfo);
+  };
+
+  return (
+    <>
+      <div className={styles.textDivider2}>
+        <div className={styles.heading}>FAQ's</div>
```

Comment:
Lets use h2 instead of div for the tag. Makes more sense as it is a heading. Also The linter problem of escapping the '